plugins/woocommerce/includes/abstracts/abstract-wc-order.php

```diff
@@ -2605,4 +2605,34 @@ public function set_cogs_total_value( float $value ) {
 			$this->set_prop( 'cogs_total_value', $value );
 		}
 	}
+
+	/**
+	 * Return the HTML to render the total Cost of Goods Sold for the order.
+	 *
+	 * @param array|null $wc_price_arg Arguments to be passed to wc_price, defaults to an array containing only the currency symbol.
+	 * @return string
+	 */
+	public function get_cogs_total_value_html( ?array $wc_price_arg = null ): string {
```

Comment:
Uh, that's an impressive line number already. What are our alternative options for hosting this rendering logic (ideally in an internal namespace so we can have more freedom to tweak this while 3PDs still can customize the HTML rendering)?

Reply:
I added this here for consistency with the `WC_Order_Item::get_cogs_value_html`, which is the equivalent for individual line items; also the class has other COGs related methods already. So while in general I agree that it's better to keep classes small and that new stuff should be added in internal namespaces, I think this is one case where consistency and readability/discoverability is more important (we could also mark this method as `@internal`, but `WC_Order_Item::get_cogs_value_html` isn't).